src/OrchardCore.Modules/OrchardCore.Tenants/Services/TenantFeatureShapeTableProvider.cs

```diff
@@ -1,10 +1,13 @@
 using OrchardCore.DisplayManagement;
 using OrchardCore.DisplayManagement.Descriptors;
 using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Modules;
 using OrchardCore.Tenants.ViewModels;
 
 namespace OrchardCore.Tenants.Services;
 
+[Feature("OrchardCore.Tenants")]
+[RequireFeatures("OrchardCore.Tenants", "OrchardCore.Features")]
```

Comment:
Idem for the not needed feature attribute.

Then maybe only`[RequireFeatures("OrchardCore.Tenants.FeatureProfiles")]`

Reply:
This class is registered here
```
    [RequireFeatures("OrchardCore.Tenants", "OrchardCore.Features")]
    public class TenantFeatureProfilesStartup : StartupBase
    {
        public override void ConfigureServices(IServiceCollection services)
        {
            services.AddScoped<IShapeTableProvider, TenantFeatureShapeTableProvider>();
        }
    }
```

so we should only need `[RequireFeatures("OrchardCore.Tenants", "OrchardCore.Features")]`

Reply:
@jtkech are you good with this PR? Can we merge it?

Reply:
@jtkech you may have forgotten about this one.

Reply:
Oops, will see tomorrow.

> so we should only need [RequireFeatures("OrchardCore.Tenants", "OrchardCore.Features")]

Hmm, maybe `OC.Tenants` is not needed here too if defined in `OC.Tenants` itself.

Don't remember the details, would need to be tried.

Reply:
Yes I agree. I'll remove it from the startup and the filter.